### DISCUSSION TITLE
🥰 V2 Plan

### DISCUSSION THREAD

**Discussion by yzh990918** [onu-ui/onu-ui#148] [Announcements]
### Description

We plan to refactor this component library, but due to our busy schedule recently, we shelved this plan, but we never gave up the idea of this plan. We very much welcome the idea of the'PR 'project. When we are done with what we are doing, we will restart the development of `V2` . All the best wishes ❤️

**Comment by wzc520pyfm** [comment]
If possible, I can help with the refactoring task. Do we have a todo list?

**Reply by wzc520pyfm** [reply]
Perhaps you could draft a refactoring guide to get more people involved?

**Reply by nei1ee** [reply]
Me too, I think the long interruption of refactoring time may not be conducive for `onu-ui`.

**Reply by yzh990918** [reply]
> Me too, I think the long interruption of refactoring time may not be conducive for `onu-ui`.

Sorry, May is so busy 😢
